README.md

```diff
@@ -14,8 +14,13 @@ composer require lmc/aerospike-cache
 ```
 
 ## Usage
+```php
+$aerospike = new \Aerospike(['hosts'=>[['addr'=>'127.0.0.1', 'port'=>3000]]]);
```

Comment:
Please format the code to make it consistent with our codestyle:
```
$aerospike = new \Aerospike(['hosts' => [['addr' => '127.0.0.1', 'port' => 3000]]]);
```

Reply:
Ok done